### ISSUE TITLE
scientific notation

### ISSUE THREAD

**Issue by gchling** [KvanTTT/MathExpressions.NET#12]
can it support scientific notation？

For example : 4.25*9.8/1e3

**Comment by KvanTTT** [comment]
No. You can check for supported syntax in the [grammar](https://github.com/KvanTTT/MathExpressions.NET/blob/master/MathExpressions.NET/MathExpr.g4).
